plugins/chat/lib/chat/guardian_extensions.rb

```diff
@@ -254,5 +254,9 @@ def can_react?
     def can_delete_category?(category)
       super && category.deletable_for_chat?
     end
+
+    def can_remove_members?(chatable)
+      is_admin? && !(chatable.is_a?(Chat::DirectMessage) && !chatable.group?)
```

Comment:
It’s probably better to rely on the `channel` API for testing this:
```rb
channel.direct_message_channel? && !channel.direct_message_group?
```

Reply:
Yes! I was looking for exactly this. 🤩